tikv-client/src/main/java/com/pingcap/tikv/meta/TiColumnInfo.java

```diff
@@ -130,7 +130,8 @@ public String getName() {
   }
 
   public boolean matchName(String name) {
-    return this.name.equalsIgnoreCase(name);
+    return this.name.equalsIgnoreCase(name)
+        || String.format("`%s`", this.name).equalsIgnoreCase(name);
```

Comment:
When will a column be quoted?

Reply:
```
CREATE TABLE `pt` (
  `id` int(11) DEFAULT NULL,
  `y` date DEFAULT NULL,
  KEY `idx_y` (`y`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin
PARTITION BY RANGE ( `id` + 1 ) (
  PARTITION p0 VALUES LESS THAN (2),
  PARTITION p1 VALUES LESS THAN (4),
  PARTITION p2 VALUES LESS THAN (6)
)
```

If we have above table schema, `partExpr` in `PartitionInfo` is "`id`". This is just a hard code version. When parser gets merged, I will rewrite this part.